samples/drivers/video/capture_to_lvgl/boards/mini_stm32h743.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_CLOCK_STM32_MCO1_SRC_HSI48=y
+CONFIG_CLOCK_STM32_MCO1_DIV=4
```

Comment:
Those symbols are now deprecated and MCO configuration should be done by device tree (see https://github.com/zephyrproject-rtos/zephyr/pull/76556)

Reply:
Thanks, @erwango. Since this version works using the "VIDEO_SW_GENERATOR", I'm removing this configuration from here, and I'll add the new MCO configuration on `:shield: weact_ministm32h7xx_ov2640`, PR https://github.com/zephyrproject-rtos/zephyr/pull/77949.